ChattoAdditions/Source/Chat Items/BaseMessage/Views/BaseMessageCollectionViewCell.swift

```diff
@@ -325,7 +336,28 @@ open class BaseMessageCollectionViewCell<BubbleViewType>: UICollectionViewCell,
         self.accessoryTimestampView.removeFromSuperview()
     }
 
+    // MARK: Selection
+
+    private let selectionIndicator = UIImageView(frame: .zero)
+
+    private func updateSelectionIndicator(with style: BaseMessageCollectionViewCellStyleProtocol) {
+        self.selectionIndicator.image = style.selectionIndicatorIcon(for: self.messageViewModel)
+    }
+
+    private lazy var selectionTapGestureRecognizer: UITapGestureRecognizer = {
```

Comment:
does it really need to be lazy? it looks like it's used in init.

Reply:
I'll fix it